src/generator/operations.ts

```diff
@@ -726,6 +731,11 @@ function createInterfaceDefinition(group: OperationGroup, imports: ImportManager
     }
     const returns = generateReturnsInfo(op, false);
     interfaceText += `\t${opName}(${getAPIParametersSig(op, imports)}) (${returns.join(', ')})\n`;
+    // Add resume LRO poller method for each Begin poller method
+    if (isLROOperation(op) && !op.extensions!['x-ms-pageable']) {
```

Comment:
Why not for `x-ms-pageable`, or will this be removed once we fix intersection of LROs and pageables?

Reply:
Yes it will be fixed once I add pagers. Here's the pending issue #362. Also I'll add that after I finish adding all the tests. It shouldnt be too complicated now that everything else is in place.